quartz/plugins/transformers/ofm.ts

```diff
@@ -114,9 +114,11 @@ const commentRegex = new RegExp(/%%(.+)%%/, "g")
 // from https://github.com/escwxyz/remark-obsidian-callout/blob/main/src/index.ts
 const calloutRegex = new RegExp(/^\[\!(\w+)\]([+-]?)/)
 const calloutLineRegex = new RegExp(/^> *\[\!\w+\][+-]?.*$/, "gm")
-// (?:^| )   -> non-capturing group, tag should start be separated by a space or be the start of the line
-// #(\w+)    -> tag itself is # followed by a string of alpha-numeric characters
-const tagRegex = new RegExp(/(?:^| )#(\p{L}+)/, "gu")
+// (?:^| )              -> non-capturing group, tag should start be separated by a space or be the start of the line
+// #(...)               -> capturing group, tag itself must start with #
+// (?:[-_\p{L}])+       -> non-capturing group, non-empty string of (Unicode-aware) alpha-numeric characters, hyphens and/or underscores
+// (?:\/[-_\p{L}]+)*)   -> non-capturing group, matches an arbitrary number of tag strings separated by "/"
+const nestedTagRegex = new RegExp(/(?:^| )#((?:[-_\p{L}])+(?:\/[-_\p{L}]+)*)/, "gu")
```

Comment:
sanity check tests using [regexr](https://regexr.com/)
![Screen Shot 2023-08-27 at 4 32 06 PM](https://github.com/jackyzha0/quartz/assets/22568643/8f5933cf-0ac4-44b7-8fcc-d73e3a43c2e8)

Reply:
oh you know what I just forgot that tags can contain numbers lol, let's add `\d` too (imo it's ok to accept tags that only contain numbers even if Obsidian doesn't allow it)

Reply:
bump on this in case you wanted to finish pushing this PR through

Reply:
Had a busy past few days, but glad this was taken to the finish line! 🙏🏼